### ISSUE TITLE
Unable to remove Built-In Entity from app through API

### ISSUE THREAD

**Issue by MushAsterion** [wit-ai/wit#2283]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
Bug with API endpoint `/entities/:entity` on method `DELETE`.

**What is the current behavior?**
Each time I send a `DELETE` request do `/entities/:entity` API endpoint for a built-in entity it throws an error while the exact same request with a non-built-in entity returns a success. Here is the error that I have:
```
{
    "error": "You can't add keywords to free-text entities.",
    "code": "unsupported-lookups"
}
```
I even tried to remove the entity by removing all of its roles and it throws the exact same error response.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Add a built-in entity to the app such as "`wit/notable_person`"
2. Send a `DELETE` request with relevant authorization to `https://api.wit.ai/entities/wit$notable_person?v=20211229`
3. You should receive the same error as above.

**What is the expected behavior?**
Named built-in entity should be removed from the app.

**If applicable, what is the App ID where you are experiencing this issue? If you do not provide this, we cannot help.**
`455899615941706` (this is a test app).

**Comment by yuzh174** [comment]
Hi @MushAsterion, thanks for reporting this issue. This seems to be a bug behavior and we will fix it soon. For now, you can use the UI button to delete it to get unblocked.

**Comment by MushAsterion** [comment]
Hey @yuzh174, thank you for the heads up, I will certainly do!

**Comment by Barbog** [comment]
Closing due to no movement on the issue. Please re-open or file a new task should the issue be persisting.
